packages/core/src/components/workflow-steps/workflow-steps.tsx

```diff
@@ -62,24 +62,22 @@ export class WorkflowSteps {
   private deselectAll() {
     const steps = this.getSteps();
     steps.forEach((element) => {
-      element.setAttribute('selected', 'false');
+      element.selected = false;
     });
   }
 
   styling() {
     let steps = this.getSteps();
     steps.forEach((element, index) => {
-      element.setAttribute('vertical', this.vertical ? 'true' : 'false');
-      element.setAttribute('clickable', this.clickable ? 'true' : 'false');
-      element.setAttribute(
-        'selected',
-        this.selectedIndex === index ? 'true' : 'false'
-      );
-      if (index === 0 && steps.length > 1)
-        element.setAttribute('position', 'first');
-      if (steps.length === 1) element.setAttribute('position', 'single');
+      element.vertical = this.vertical;
+      element.clickable = this.clickable;
+      element.selected = this.selectedIndex === index ? true : false;
+
+      if (index === 0 && steps.length > 1) element.position = 'first';
```

Comment:
I feel like this whole block of conditionals can be simplified by grouping them a bit, e.g.:

_if (steps.length === 1) {
 [..]
  return;
}

if (steps.length > 1) {
 
} else {
}_